job_executor/app.py

```diff
@@ -144,32 +186,24 @@ def main():
     try:
         while True:
             time.sleep(5)
+
+            job_dict = query_for_jobs()
+            queued_worker_jobs = job_dict["queued_worker_jobs"]
```

Comment:
This lookup will fail if the database is paused, as query_for_jobs returns None in that case 👀